cmd/cosign/webhook/kodata/LICENSE

```diff
@@ -0,0 +1 @@
+../../../../LICENSE
```

Comment:
Question: why do we need this here ?

Reply:
Could we add these `kodata` files to `.gitignore` ?

Reply:
They aren't generated, and they are generally checked in because you want the published images to contain the metadata.  If there were a way of generating these, we could conceivably use `.gitignore`, but `ko` isn't going to create these (`kodata` is its convention, not the rest)